runners/flink/flink_runner.gradle

```diff
@@ -134,6 +134,7 @@ dependencies {
   compile project(":runners:java-fn-execution")
   compile project(":runners:java-job-service")
   compile project(":sdks:java:extensions:google-cloud-platform-core")
+  compile project(":sdks:java:io:kafka")
```

Comment:
Maybe worth to add the TODOs to remove this once SDF is fixed because it does not make sense that runners depend on IOs at least not for non test scopes

Reply:
Good point! Added TODO all build files.

Reply:
> One way I'm thinking about is to always populate beam_fn_api_use_deprecated_read and use_deprecated_read when use_sdf_read is not specified.

I think the method mentioned above might be a better way to do this since we could avoid the io dependency on runners.

Reply:
Usually populating experiments from runner is the least preferred one for me. With applying override utils from transform directly, we will have a centralized place to change the logic if needed.

Reply:
Yeah, it makes sense. But I still prefer to not introduce the additional io dependency to runners given that this change is temporary and will be reverted soon once we fix the performance issue (I'm not totally against the current implementation though so it's good to go if other folks don't have any preference).

Reply:
I really want this to move on but I have to say that I agree with @ihji, another reason is that adding the KafkaIO dependency to almost all runners will bring a lot of the extra baggage that KafkaIO has including extra confluent and spring dependencies that will leak into the user system's classpath and can produce conflicts.

Reply:
Changed to populating experiments directly.